docs/baseband/tutorials/getting_started.rst

```diff
@@ -256,25 +256,66 @@ VDIF file's headers are of class::
 
 and so its attributes can be found `here <baseband.vdif.header.VDIFHeader3>`.
 
-Opening Specific Threads/Channels From Files
---------------------------------------------
+Opening Specific Components of the Data
```

Comment:
Note somewhere that it is better to put `slice` than fancy indices in `subset`

Reply:
And put somewhere that `squeeze` is immutable.

Reply:
Maybe for the title of the section, state `Reading Specific Components..` or `Opening a File for Reading Only Specific Components`